Lib/test/test_tools/test_makefile.py

```diff
@@ -67,11 +67,14 @@ def test_makefile_test_folders(self):
                 )
                 used.append(relpath)
 
+        unique_used = set(used)
```

Comment:
I would prefer to override the used variable here, since it doesn't seem to be used as a list anyway.

```suggestion
        used = set(used)
```

Reply:
I'll just make it a set from the beginning